BHoM_Adapter/AdapterActions/Push.cs

```diff
@@ -94,10 +94,17 @@ public virtual List<object> Push(IEnumerable<object> objects, string tag = "", P
             List<Tuple<Type, PushType, IEnumerable<object>>> orderedObjects;
 
             if(m_AdapterSettings.HandleDependencies)
+            {
                 orderedObjects = Engine.Adapter.Query.GetDependencySortedObjects(objectsToPush, pushType, this);
+                if (m_AdapterSettings.HandlePriorities)
+                    orderedObjects = Engine.Adapter.Query.GetPrioritySortedObjects(orderedObjects, pushType, this);
```

Comment:
If we keep the HandlePriorty settings boolean, which actually might be a good shout to keep, then these lines should be moved down to after this if/else statement, to make it possible to use the priority types, even if HandleDependecnies is false.
```suggestion
```

Reply:
@IsakNaslundBh , thanks for this.
Change incorporated in the code.